ote/src/cljs/ote/views/service_viewer.cljs

```diff
@@ -493,32 +531,362 @@
+       [common-ui/information-row-with-option
+        (tr [:service-viewer :assistance-phone])
+        requirements-telephone false]]]
+     [spacer]]))
+
+(defn- parking-facilities [title data]
+  [:section
+   [info-sections-2-cols title
+    [:div
+     (doall
+       (for [row data
+             :let [facility (::t-service/parking-facility row)]]
+         ^{:key (str row facility)}
+         [common-ui/information-row-with-option
+          (tr [:field-labels :parking :ote.db.transport-service/parking-facility])
+          (tr [:enums ::t-service/parking-facility facility]) false]))]
+    [:div
+     (doall
+       (for [row data
+             :let [capacity (::t-service/capacity row)]]
+         ^{:key (str row capacity)}
+         [common-ui/information-row-with-option
+          (tr [:field-labels :parking :ote.db.transport-service/capacity])
+          capacity false]))]]
+   [spacer]])
+
+(defn- charging-points [title description]
+  [:section
+   [info-sections-1-col title
+    [:div
+     [information-row-with-selection (tr [:common-texts :description]) (format-descriptions description) true]]]
+   [spacer]])
+
+(defn- accessibility-info [title accessibility accessibility-description accessibility-info-url information-service-accessibility]
+  [:section
+   [info-sections-1-col title
+    [:div
+     [common-ui/information-row-with-option
+      (tr [:service-viewer :accessibility])
+      (string/join ", " (map (fn [key] (tr [:enums ::t-service/accessibility key])) accessibility))
+      true]
+     [common-ui/information-row-with-option (tr [:field-labels :terminal :ote.db.transport-service/information-service-accessibility])
+      (string/join ", " (map (fn [key] (tr [:enums ::t-service/information-service-accessibility key])) information-service-accessibility))
+      true]
+     [common-ui/information-row-with-option (tr [:service-viewer :accessibility-website]) accessibility-info-url true]
+     [information-row-with-selection (tr [:field-labels :transport-service-common ::t-service/accessibility-description]) (format-descriptions accessibility-description) true]]]
+   [spacer]])
+
+(defn- parking-restrictions [title data]
+  (let [parking-restricted? (or (not (empty? data)) false)
+        restriction-time-type (cond
+                                (and
+                                  (not (nil? (:minutes data)))
+                                  (> (:minutes data) 0)) :minutes
+                                (and
+                                  (not (nil? (:hours data)))
+                                  (> (:hours data) 0)) :hours
+                                (and
+                                  (not (nil? (:days data)))
+                                  (> (:days data) 0)) :days
+                                :default :hours)
+        restriction-value (restriction-time-type data)]
```

Comment:
Isn't `restrisction-time-type` a symbol which gets bound in `let` initialisation, so instead of `restriction-value` code could refer directly `restrisction-time-type`.
(Or if not, then `restrisction-time-type?` should be declared as `fn`)

https://clojuredocs.org/clojure.core/some_q

Reply:
BTW `(some? v)` is equals to `(not (empty? v)`